api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/highlight.md

```diff
@@ -5,7 +5,7 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to lighten the series when a user points to it.
```

Comment:
`when a user points to it` With what? A finger?

Reply:
With a mouse pointer

Reply:
```suggestion
Specifies whether to lighten the series when a user hovers the mouse pointer over it.
```